docs/source/index.rst

```diff
@@ -0,0 +1,28 @@
+.. AugLy documentation master file, created by
+   sphinx-quickstart on Tue Dec  7 18:02:52 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to AugLy's documentation!
+=================================
+
+.. toctree::
+   :maxdepth: 4
+   :caption: Contents:
+
+   README.md
+   README_audio.md
+   README_image.md
+   README_video.md
+   README_text.md
+   augly.audio
+   augly.image
+   augly.video
+   augly.text
```

Comment:
does it make sense for this to be (i also alphabetized -- text should come before video because `t` is before `v`:
```suggestion
   README_audio.md
   augly.audio
   README_image.md
   augly.image
   README_text.md
   augly.text
   README_video.md
   augly.video
```